hipscat/util.py

```diff
@@ -257,13 +316,48 @@ def catalog_columns_selector_withdtype(cat_md, cols):
 
 
 def establish_pd_meta(c1_cols, c2_cols):
+    '''
+        the return type from cross_match routine metadata columns
+    '''
     colnames = []
     colnames.extend(c1_cols)
     colnames.extend(c2_cols)
     colnames.extend(['hips_k', 'hips_pix', '_DIST'])
     return colnames
 
 
+def rename_meta_dict(c1_cols, c2_cols, suffix='_2'):
```

Comment:
this has been implemented. You can't do 'prefix.column' since it will try to assume that the prefix will inherit the column name when attempting to do analysis. I implemented 'prefix_column' which works in dask api analysis